package.json

```
@@ -9,6 +9,10 @@
     "lint": "eslint . --max-warnings 0",
     "build": "tsc",
     "test": "NODE_ENV=test NODE_OPTIONS='--experimental-vm-modules --trace-warnings' jest  --coverage",
+    "db-migrate": "NODE_ENV=development node src/run-migrations.js up",
+    "db-downgrade": "NODE_ENV=development node src/run-migrations.js down",
+    "db-test-migrate": "NODE_ENV=test node src/run-migrations.js up",
+    "db-test-downgrade": "NODE_ENV=test node src/run-migrations.js down",
```

When adding tests I often came across the problem that the teardown was not called correctly, which led to the setup failing upon restarting the test suite.

I've added these pair of scripts so that one can manually clear the test database using `npm run db-test-downgrade` when that happens.

---

Could we match the syntax in actual used to group scripts here?

```suggestion
    "db:migrate": "NODE_ENV=development node src/run-migrations.js up",
    "db:downgrade": "NODE_ENV=development node src/run-migrations.js down",
    "db:test-migrate": "NODE_ENV=test node src/run-migrations.js up",
    "db:test-downgrade": "NODE_ENV=test node src/run-migrations.js down",
```